data/campaigns/Eastern_Invasion/scenarios/11_Captured.cfg

```diff
@@ -231,7 +231,9 @@
 
         {RECALL_XY Dacyn 28 13}
         {MODIFY_UNIT id=Dacyn facing se}
+#ifndef EASY
         {MODIFY_UNIT id=Dacyn hitpoints 7}
```

Comment:
maybe instead of `#ifndef EASY` do `{ON_DIFFICULTY 47 27 7}` here? If he hasn't advanced to a Mage of Light yet, the overheal will just cap him off at his previous max HP...

Reply:
I worry that 27hp would mislead players, thinking they can / are supposed to use Dacyn to fight, but not having enough HP to succeed.

IMO 47 hp lets Dacyn fight normally, and 7 hp is low enough that (hopefully) players get the idea he needs to retreat.

Reply:
ok, maybe at least just add a comment about how the "7" was chosen, then?